azure-pipelines.yml

```diff
@@ -25,10 +25,9 @@ steps:
   displayName: 'Build debug'
 
 - script: |
-   cargo test --locked --lib -- -Z unstable-options --report-time  --format junit 1> $(System.DefaultWorkingDirectory)\TEST-rust.xml
-   echo content of xml
-   type $(System.DefaultWorkingDirectory)\TEST-rust.xml
-   echo end
+   cargo install junit-test
```

Comment:
1.70 removed the ability to use unstable options in test, so use this alternative